backend/redis/conn_mgr.c

```diff
@@ -402,12 +428,200 @@ dbBE_Redis_connection_recoverable_t dbBE_Redis_connection_mgr_conn_recover(
     } // if( rec != NULL )
     else
       ++unbroken;
-  } // for( c=0; c < DBBE_REDIS_MAX_CONNECTIONS; ++c )
+  } //   for( c=0; (c < DBBE_REDIS_MAX_CONNECTIONS) && (unbroken+recovered <= conn_mgr->_connection_count); ++c )
```

Comment:
in line 268 the `for` condition has `unbroken+recovered < conn_mgr->_connection_count`. Which one is the right one?

Reply:
the actual code is the right one. The comment is a copy-paste from a previous version that didn't get updated. I should either adjust or remove those comments. Good catch.